api/app/reviews/api.py

```diff
@@ -294,6 +332,28 @@ def get(self):
 
         return ReviewResponseUser(review_form, response, 0, args['language'], review_response=review_response)
 
+class ResponseReviewEventAdminAPI(restful.Resource):
+    @event_admin_required
+    def get(self, event_id):
+        parser = reqparse.RequestParser()
+        parser.add_argument('response_id', type=int, required=True)
+        parser.add_argument('language', type=str, required=True)
+        args = parser.parse_args()
+
+        response_id = args['response_id']
+
+        review_form = review_repository.get_review_form(event_id)
+        if review_form is None:
+            return REVIEW_FORM_NOT_FOUND
+
+        response_reviews = (review_repository.get_all_review_responses_by_response(review_form.id, response_id))
+        serialized_reviews =  [
+            ReviewResponseDetailListAPI._serialise_review_response(response, args['language'])
+            for response in response_reviews
+        ]
+        return marshal(ReviewResponses(review_form, serialized_reviews, args['language']), response_review_fields), 201

```

Comment:
The status code to return for a successful GET is 200 ("OK"), not 201 which stands for "CREATED" and would usually be used in a POST request.